src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -66,14 +66,28 @@ public function testTabsInYaml()
             try {
                 $this->parser->parse($yaml);
 
-                $this->fail('YAML files must not contain tabs');
+                $this->fail('YAML files must not contain tabs as indentation');
             } catch (\Exception $e) {
-                $this->assertInstanceOf(\Exception::class, $e, 'YAML files must not contain tabs');
-                $this->assertEquals('A YAML file cannot contain tabs as indentation at line 2 (near "'.strpbrk($yaml, "\t").'").', $e->getMessage(), 'YAML files must not contain tabs');
+                $this->assertInstanceOf(\Exception::class, $e, 'YAML files must not contain tabs as indentation');
+                $this->assertEquals('A YAML file cannot contain tabs as indentation at line 2 (near "'.strpbrk($yaml, "\t").'").', $e->getMessage(), 'YAML files must not contain tabs as indentation');
             }
         }
     }
 
+    public function testTabsAsSeparationBetweenTokensInYaml()
+    {
+        // test tabs as separation between tokens
+        $yamls = [
+            'foo:	bar', // tab after colon
+            'foo: 	bar', // space and tab after colon
```

Comment:
I suggest writing those using a double-quoted string and a `\t` escape sequence to put the tab character. It will make the code a lot more readable (and the comments describing the strings won't bee needed anymore).

Reply:
Thanks for the tip, I changed it in https://github.com/symfony/symfony/pull/40514/commits/463a1c9948e978c1be7fa73c652924b291943660